src/liballoc/boxed.rs

```diff
@@ -527,6 +526,7 @@ impl<I: ExactSizeIterator + ?Sized> ExactSizeIterator for Box<I> {}
 /// ```
 #[rustc_paren_sugar]
 #[unstable(feature = "fnbox", reason = "Newly introduced", issue = "0")]
+#[fundamental]
```

Comment:
Just slip that in there :)

This is probably prudent, and technically breaking but it's unstable anyway.